service/src/main/java/bio/terra/workspace/app/controller/Alpha1ApiController.java

```diff
@@ -200,7 +195,7 @@ public ApiResourceUnion apiResourceFromWsmResource(
             {
               ControlledAiNotebookInstanceResource resource =
                   controlledResource.castToAiNotebookInstanceResource();
-              union.gcpAiNotebookInstance(resource.toApiResource(gcpProjectId));
+              union.gcpAiNotebookInstance(resource.toApiResource());
```

Comment:
This is definitely an improvement. It was awkward having to supply that project ID separately.